reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientOperations.java

```diff
@@ -122,6 +122,7 @@ class HttpClientOperations extends HttpOperations<NettyInbound, NettyOutbound>
 	Consumer<HttpClientRequest> redirectRequestConsumer;
 	HttpHeaders previousRequestHeaders;
 	BiConsumer<HttpHeaders, HttpClientRequest> redirectRequestBiConsumer;
+	Throwable unprocessedOutboundError;
```

Comment:
Do we need to handle this in the `copy` constructor?

Reply:
I see that currently, only the [WebsocketClientOperations](https://github.com/reactor/reactor-netty/blob/main/reactor-netty-http/src/main/java/reactor/netty/http/client/WebsocketClientOperations.java#L68) constructor is using the copy constructor, but I wonder if it a good idea to propagate any  unprocessedOutboundError in the newly created WebsocketClientOperations instance ?
for the moment, I'm unsure if it's needed to handle the `unprocessedOutboundError`in the copy constructor.
Are you thinking about a potential problem ?

Reply:
okay, will do that, thanks.

Reply:
added a comment in https://github.com/reactor/reactor-netty/pull/2937/commits/40e200d6bdc4d778b59b8b671d9687752fd7fa50